indra/cmake/00-Common.cmake

```diff
@@ -103,6 +103,11 @@ if (WINDOWS)
     string(REPLACE "/Zi" "/Z7" CMAKE_C_FLAGS_RELWITHDEBINFO "${CMAKE_C_FLAGS_RELWITHDEBINFO}")
     string(REPLACE "/Zi" "/Z7" CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO}")
   endif()
+
+  # workaround for github runner image breakage:
+  # https://github.com/actions/runner-images/issues/10004#issuecomment-2153445161
+  # can be removed after the above issue is resolved and deployed across GHA
+  add_compile_definitions(_DISABLE_CONSTEXPR_MUTEX_CONSTRUCTOR)
```

Comment:
It seems like the github runner incident may have been resolved: https://www.githubstatus.com/incidents/lfrlwdg67fn8

Reply:
Not sure about that, I'm still getting the same errors on the branch without this change.